app/src/main/res/values/strings.xml

```diff
@@ -1442,7 +1444,7 @@
 
     <!-- This string is displayed in the description of the Voice Search samples collect dialog. That dialog is shown the first time
          a user tries to use the voice search. -->
-    <string name="voice_samples_collect_dialog_description2">In order to improve our voice recognition and other services, we need to collect voice samples for research. We store your data securely and without identifying information. You’ll always be able to use voice search, even if you don’t allow collection.&lt;a href="privacy">Learn More&lt;/a></string>
+    <string name="voice_samples_collect_dialog_description2"><![CDATA[In order to improve our voice recognition and other services, we need to collect voice samples for research. We store your data securely and without identifying information. You’ll always be able to use voice search, even if you don’t allow collection. <a href="">Learn More</a>]]></string>
```

Comment:
```suggestion
    <string name="voice_samples_collect_dialog_description2"><![CDATA[In order to improve our voice recognition and other services, we need to collect voice samples for research. We store your data securely and without identifying information. You’ll always be able to use voice search, even if you don’t allow collection. <a href="privacy">Learn More</a>]]></string>
```

Reply:
The added ` ` is OK w/out ID change, but the link got lost in this one still.

Reply:
That one doesn't require a link, it's handled internally, so it should be good.